### ISSUE TITLE
Low test accuracy with resnet 110 cifar10

### ISSUE THREAD

**Issue by XinLiuNvidia** [keras-team/keras#10632]
I train a few resnet v2 110 model with cifar10, and only get 92.x% accuracy on the test set. In the table, it shows test accuracy is about 93.x%. I wonder how to get that?

**Comment by XinLiuNvidia** [comment]
I tried the default learning rate, regularization in cifar10_resnet.py, and a few other hyper parameters, still get 92.x% on test set.

**Comment by roatienza** [comment]
It should be replicable if you run GPU and epochs > 120

**Comment by XinLiuNvidia** [comment]
@roatienza do you mean use the default parameters, n=12, version=2, I can reproduce the 93.x% on the test set by running cifar10_resnet10.py? or do I need to change any hyper parameters? Could you share your hyper parameter settings to reproduce the result?

**Comment by roatienza** [comment]
Yes. `n=12`, `version=2`. The best accuracy is about 93.15% as shown on the table.

**Comment by fuzzythecat** [comment]
@XinLiuNvidia  increasing `batch_size` to 128 got me to 93.x% accuracy on the test set.

**Comment by XinLiuNvidia** [comment]
@fuzzythecat thank you very much! After sync to the latest keras version, I'm able to reproduce the accuracy 93.x% on the test set.
